packages/core/src/integrations/functiontostring.ts

```diff
@@ -22,10 +22,12 @@ export class FunctionToString implements Integration {
     // eslint-disable-next-line @typescript-eslint/unbound-method
     originalFunctionToString = Function.prototype.toString;
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    Function.prototype.toString = function (this: WrappedFunction, ...args: any[]): string {
-      const context = getOriginalFunction(this) || this;
-      return originalFunctionToString.apply(context, args);
-    };
+    try {
```

Comment:
Could you add a comment here for why this try-catch exists? For future reference :) Would also be nice to have an example when/why this may be frozen.

Reply:
thanks, that's perfect!